csharp/test/Ice/udp/Client.cs

```diff
@@ -1,42 +1,37 @@
 // Copyright (c) ZeroC, Inc.
 
-namespace Ice
+namespace Ice.udp;
+
+public class Client : global::Test.TestHelper
 {
-    namespace udp
+    public override async Task runAsync(string[] args)
     {
-        public class Client : global::Test.TestHelper
-        {
-            public override void run(string[] args)
-            {
-                var properties = createTestProperties(ref args);
-                properties.setProperty("Ice.Warn.Connections", "0");
-                properties.setProperty("Ice.UDP.SndSize", "16384");
-                using (var communicator = initialize(properties))
-                {
-                    AllTests.allTests(this);
```

Comment:
That was the root bug. allTests is async but we didn't await it since returns void.

Reply:
Everything else is reformatting!